apis/core.oam.dev/common/types.go

```diff
@@ -115,11 +114,7 @@ type Terraform struct {
 	// +kubebuilder:validation:Enum:=hcl;json;remote
 	Type string `json:"type,omitempty"`
 
-	// Path is the sub-directory of remote git repository. It's valid when remote is set
-	Path string `json:"path,omitempty"`
-
-	// ProviderReference specifies the reference to Provider
-	ProviderReference *types.Reference `json:"providerRef,omitempty"`
+	v1beta1.BaseConfigurationSpec `json:",inline"`
```

Comment:
do we keep the compatiblity here？

Reply:
Yes, the CRD didn't change and I have tested VPC with remote git repo. And I will port e2e tests from https://github.com/oam-dev/terraform-controller/blob/master/.github/workflows/e2e-test.yml to this repo once it got stable.